src/backend/routes/cheap_meals.js

```diff
@@ -0,0 +1,25 @@
+// Respond with the json for all the meals (including it's reviews) 
+// that are cheap (you define what a cheap meal is)
+
+const mealsArray = require("../data/meals.json");
+const reviewsArray = require("../data/reviews.json");
+// const reservationsArray = require("../data/reservations.json");
+
+
+const getcheapmeal = function(request, response){
+    
+    const mealreviewsArray = mealsArray.map(mealitem=>{
+        if(mealitem.price <= 50){
+            mealitem.reviews = reviewsArray.filter(element => mealitem.id === element.mealId)
+               return mealitem;
+            }else{
+                return
+                }
+
+    })

```

Comment:
The indentation here could be improved a bit, it is a bit difficult to see which curly braces match with which.
But more importantly your approach is slightly incorrect because you end up either returning a meal object if the meal is cheap, or you return `undefined`. So that produces a list of meals and `undefined` values but you only want to return meals.